### ISSUE TITLE
Datatyp error by value 2.47812E-05

### ISSUE THREAD

**Issue by ortnever** [zazuko/cube-creator#1283]
Is there a range of allowed values for the decimal type ?
I have a value of 2.47812E-05 in the project UBD000501 Treibhausgasemissionen nach Sektoren (CO2-Verordnung) and the transformation fails.

In my CSV the value appears in the non-scientific notation, i.e. 0.000024782 but the transformation blocks on this line.
https://gitlab.ldbar.ch/pipelines/cube-creator/-/jobs/34003

![image](https://user-images.githubusercontent.com/79834090/187232274-ace11358-b312-44f6-84b3-7a0159e8a913.png)

**Comment by ortnever** [comment]
Regardless of the allowed value range, is scientific notation, for example, E-05 accepted by the transformation?

**Comment by tpluscode** [comment]
The value is the same, regardless of the notation. This should not be failing.

**Comment by Rdataflow** [comment]
@tpluscode might be related to datatyp. 

IIUC *scientific notation* is only available for `xsd:double`

cc @l00mi

**Comment by tpluscode** [comment]
You are right, scientific notation is reserved for floating point numbers (float and double). I spent some time trying to find a possible bug in the assumption that the pipeline incorrectly transforms that number

Instead, I found that row 622 does in fact contain `"2.47812E-05"` in the value column. It must have been changed when exporting source data to CSV. The pipeline is correct to signal a problem.

<img width="251" alt="Screenshot 2022-09-05 at 09 58 11" src="https://user-images.githubusercontent.com/588128/188398405-3a7572c1-6a77-40ed-a904-406bf5586766.png">

**Comment by l00mi** [comment]
For context: @ortnever and @Rdataflow we did provide the datatypes xsd:float and xsd:double. But this was confusing for the data providers (mentioned by @FabianCretton). As 99% are covered by xsd:decimal we only kept this datatype. In this case I propose to fix the data source. If the need arises we can include these datatypes again.

**Comment by ortnever** [comment]
Thank you for your research and sorry for the confusion. I told you that in my CSV the value appeared in the non-scientific notation and it was the case. Unfortunately the file was not really replaced because I did not notice that one column had a different name.

It is clearly written by upload that the column names should not be changed. However, if this is the case, the user do not receive any error message and everything happens as if the file had been replaced. This is dangerous in my opinion, the user should receive an error message. Wasn't this the case before?
